### DISCUSSION TITLE
Best practices while using html clipping

### DISCUSSION THREAD

**Discussion by arunnbabu81** [buggyj/tiddlyclip-plugin#74] [General]
How large can be a html tiddler without affecting loading time.
Is it always better to open html tiddlers in ckeditor for better performance

**Comment by buggyj** [comment]
there is no time difference when loading a TW with html tiddlers (they take up the same space as tiddlywiki tiddlers). There is a small delay the first time the ckeditor is loaded, but for most modern computers this is not noticed. I have not experimented with very large html tiddlers with ckeditor, most of my clips are not very big(less than 100 lines).  typically I add notes to my clips with the ckeditor plugin (this is why I wrote it) and there is no other practical way of doing this within a tiddltywiki.

**Reply by arunnbabu81** [reply]
Today i tried to clip an journal article using `html `and `select ` features of tiddlyclip and it was a big tiddler with many images. I noticed a few seconds delay in opening the tiddler into story river on clicking the tiddler link and also while opening that tiddler in edit mode. Once opened it was working fine.

**Reply by arunnbabu81** [reply]
> typically I add notes to my clips with the ckeditor (this is why I wrote it) and there is no other practical way of doing this within a tiddltywiki.

how do you add notes to your html clips using ckeditor?

**Comment by arunnbabu81** [comment]
Do you use any specific ckeditor plug ins for annotating the clipped html tiddlers. Can you share a sample screenshot of how you annotate them?

**Reply by buggyj** [reply]
# I create a hover using the link button:

![image](https://user-images.githubusercontent.com/2855795/153769830-d7f504fc-a490-49e3-8d92-0ba3b406fe9b.png)

<hr>

![image](https://user-images.githubusercontent.com/2855795/153769936-1f922d1f-daad-4b6b-b63f-9fd81ceccf94.png)

<hr>

![image](https://user-images.githubusercontent.com/2855795/153770074-18539c58-7a56-40b5-9d3f-9e45d50ad734.png)

<hr>

![image](https://user-images.githubusercontent.com/2855795/153770303-f446e525-654b-4aed-ac5e-8d5e8f477c7b.png)
